packages/components/src/components/selects/Select/index.native.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
I think we don't need Async Select in RN, so this file can be deleted.
I would also do a native file for exported components in `src/index.native.tsx`, because we have less exported components in RN.

Reply:
removed